src/frontend/js/modules/i18n/i18n-loader.service.js

```diff
@@ -23,7 +23,8 @@
       };
 
       function getCatalogs() {
-        return $http.get('/api/i18n').then(function(res) {
+        const url = window.openpaas && window.openpaas.OPENPAAS_API_URL ? window.openpaas.OPENPAAS_API_URL + '/api/i18n' : '/api/i18n';
```

Comment:
```suggestion
        const url = (window.openpaas && window.openpaas.OPENPAAS_API_URL || '') + '/api/i18n';
```

Why this request does not use configured `BaseURL` above?

Reply:
thanks. I updated it